Foundation/NSString.swift

```diff
@@ -1149,7 +1455,7 @@ extension NSString {
         if !getBytes(nil, maxLength: Int.max - 1, usedLength: &numBytes, encoding: enc, options: [], range: theRange, remaining: nil) {
             throw NSError(domain: NSCocoaErrorDomain, code: CocoaError.fileWriteInapplicableStringEncoding.rawValue, userInfo: [
                 NSURLErrorKey: dest,
-            ])
+                ])
```

Comment:
Is this indentation expected? I thought it looked better before, where it was aligned with the 'throw'